### ISSUE TITLE
Missing Icons Activity Bar

### ISSUE THREAD

**Issue by scrossan-crown** [microsoft/vscode#76527]
Version: 1.36.0-insider (user setup)
Commit: 0f3794b38477eea13fb47fbe15a42798e6129338
Date: 2019-07-03T08:00:29.662Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18922
Steps to Reproduce:

1. Open Visual Studio Code Insiders
2. See blank spaces on the activity bar which are extensions with missing icons

![image](https://user-images.githubusercontent.com/30405149/60607332-f3b65e00-9db4-11e9-8ce3-72f5e2bc3db8.png)

![image](https://user-images.githubusercontent.com/30405149/60607274-e26d5180-9db4-11e9-989a-48dc6ca6d579.png)

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by misolori** [comment]
I tested this with the same version of Insiders and on Windows and am not able to reproduce:

<img width="469" alt="image" src="https://user-images.githubusercontent.com/35271042/60617596-e7b4a700-9d88-11e9-9176-e5cb9c0c5f30.png">


Version: 1.36.0-insider (system setup)
Commit: 0f3794b38477eea13fb47fbe15a42798e6129338
Date: 2019-07-03T08:00:29.662Z
Electron: 4.2.5
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

@scrossan-crown have you tried restarting VS Code and/or re-installing the extensions?

@bpasero @sandy081 not sure if this could be an activity bar cache issue (if we still do that)?

**Comment by scrossan-crown** [comment]
I've just tried restarting VS Code Iniders and re-installing the extensions but it hasn't helped. I have noticed one other thing that it's only an issue when I'm using the remote development (WSL) extension but I don't get the same issue using the stable (non insiders) version of vscode.

![image](https://user-images.githubusercontent.com/30405149/60644979-69f7a680-9e2f-11e9-9e5d-be5774907740.png)

![image](https://user-images.githubusercontent.com/30405149/60645090-afb46f00-9e2f-11e9-93e1-617fb8e32ce8.png)

![image](https://user-images.githubusercontent.com/30405149/60645294-3e28f080-9e30-11e9-86fa-d674437900e6.png)

**Comment by scrossan-crown** [comment]
I've found further errors in the dev tools.
![image](https://user-images.githubusercontent.com/30405149/60652928-f3fc3b00-9e40-11e9-9a4e-85cf31b3d17e.png)

**Comment by bpasero** [comment]
+ @jrieken @mjbvz @alexandrudima

**Comment by jrieken** [comment]
Should be unrelated to the work in #75061 since that only loads when the web client loads... Maybe some issue with main-side interceptor? I am open to fix this with the web approach tho...

**Comment by bpasero** [comment]
Yeah going with the new approach is a good idea, but I wonder what could have changed. If this was reproducing on all setups we would have more people hitting it.

@scrossan-crown do you see this on all workspaces you open or only specific scenarios?

**Comment by scrossan-crown** [comment]
It happens on all workspaces or even without a workspace

**Comment by jrieken** [comment]
This is likely the same as https://github.com/microsoft/vscode-remote-release/issues/687
